diff --git a/apps/isaaclab.python.kit b/apps/isaaclab.python.kit
index 2435019..49610dd 100644
--- a/apps/isaaclab.python.kit
+++ b/apps/isaaclab.python.kit
@@ -38,7 +38,7 @@ keywords = ["experience", "app", "usd"]
 
 # Isaac Sim Extra
 "isaacsim.asset.importer.mjcf" = {}
-"isaacsim.asset.importer.urdf" = {version = "2.4.31", exact = true}
+"isaacsim.asset.importer.urdf" = {}
 "omni.physx.bundle" = {}
 "omni.physx.tensors" = {}
 "omni.replicator.core" = {}
